core/src/main/java/com/linecorp/armeria/server/cors/CorsConfig.java

```diff
@@ -208,13 +208,12 @@ public String toString() {
      * It's value must be generated when the response is generated, hence will be
      * different for every call.
      */
-    static final class InstantValueSupplier implements Supplier<Instant> {
-
-        static final InstantValueSupplier INSTANCE = new InstantValueSupplier();
+    enum TimestampSupplier implements Supplier<String> {
+        INSTANCE;
 
         @Override
-        public Instant get() {
-            return Instant.now();
+        public String get() {
+            return HttpTimestampSupplier.currentTime();
```

Comment:
Perhaps we could remove this class and just use `HttpTimestampSupplier::currentTime`?